lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/aplinear/APLSummarizer.java

```diff
@@ -53,12 +52,10 @@ public void process(DataFrame input) throws Exception {
         encoder = new AttributeEncoder();
         encoder.setColumnNames(attributes);
         long startTime = System.currentTimeMillis();
-        List<int[]> encoded = encoder.encodeAttributes(
-                input.getStringColsByName(attributes)
-        );
+        int[][] encoded = getEncoded(input.getStringColsByName(attributes), input);
         long elapsed = System.currentTimeMillis() - startTime;
-        log.debug("Encoded in: {}", elapsed);
-        log.debug("Encoded Categories: {}", encoder.getNextKey());
+        log.info("Encoded in: {}", elapsed);
```

Comment:
Nit: can we change this to "Encoded in: {} ms"—it's much more informative when people actually look at the logs